api/models.py

```diff
@@ -1,3 +1,39 @@
+import binascii
+import os
+
+from django.conf import settings
 from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext_lazy as _
+
+
+@python_2_unicode_compatible
+class Token(models.Model):
+    key = models.CharField(_("Key"), max_length=40, primary_key=True)
+    ip_stricted = models.BooleanField(null=False, default=True, blank=False)
+    created = models.DateTimeField(_("Created"), auto_now_add=True)
+
+    class Meta:
+        abstract = 'api' not in settings.INSTALLED_APPS
```

Comment:
I thought nobody should be able to use this if this is not in the INSTALLED_APPS.
But, I`m not sure.